test/ffmpeg/version_test.rb

```diff
@@ -1,29 +1,36 @@
 require 'test_helper'
 
-class Ffprober::Ffmpeg::VersionTest < Minitest::Test
-  class FakeExec
-    attr_writer :output
-    def ffprobe_version_output
-      @output
-    end
-  end
+module Ffprober
+  module Ffmpeg
+    class VersionTest < Minitest::Test
+      class FakeExec
+        attr_writer :output
+        def ffprobe_version_output
+          @output
+        end
+      end
 
-  def test_version_output
-    exec = FakeExec.new
+      def test_version_output
+        exec = FakeExec.new
 
-    Dir.new("test/assets/version_outputs").each do |entry|
-      next if [".", "..", ".DS_Store"].include?(entry)
-      os, expected_version = entry.split("-")
+        Dir.new('test/assets/version_outputs').each do |entry|
+          next if ['.', '..', '.DS_Store'].include?(entry)
+          _os, expected_version = entry.split('-')
 
-      version_output = File.read("test/assets/version_outputs/" + entry)
-      exec.output = version_output
+          version_output = File.read('test/assets/version_outputs/' + entry)
+          exec.output = version_output
 
-      ffprobe_version = Ffprober::Ffmpeg::Version.new(exec)
+          ffprobe_version = Ffprober::Ffmpeg::Version.new(exec)
 
-      if expected_version == "nightly"
-        assert ffprobe_version.nightly?
-      else
-        assert_equal Gem::Version.new(expected_version.tr("_", ".")), ffprobe_version.version
+          if expected_version == 'nightly'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.